entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -48,11 +50,26 @@ class EntityOperationHandler(
             }
         )
 
+        val extractedEntities = JsonUtils.parseListOfEntities(body)
```

Comment:
would be great to avoid this extra parsing (already done a few lines above when parsing to NGSI-LD entities)